test/system/200-pod.bats

```diff
@@ -339,6 +339,36 @@ EOF
     run_podman rmi $infra_image
 }
 
+@test "podman pod create should write its pod id to the file specified by --out" {
+    outfile=${PODMAN_TMPDIR}/out-results
+    local infra_image="k8s.gcr.io/pause:3.5"
+    local pod_name="$(random_string 10 | tr A-Z a-z)"
+
+    # copypasta
+    run_podman --out $outfile pod create --name $pod_name --infra-name "$infra_name" --infra-image "$infra_image"
+    is "$output" "" "output from pod create should be empty"
+    contents="$(<$outfile)"
+
+    # next we need to get the pod id from the name that was used
+    run_podman pod inspect $pod_name --format '{{.Id}}'
+    is "$output" "$contents" "output should the pod id that was emitted during creation"
+
+    # enumerate the pods and find the one matching our name
+    run_podman --out $outfile pod ls --format json
+    is "$output" "" "output from pod ls should be empty"
+
+    # use the output file to select an entry matching the desired id..
+    jq -r --arg id $contents '.[] | select(.Id == $id) | .Name' $outfile
+    is "$output" "$pod_name" "pod name for the created pod with the retrieved id"
```

Comment:
ah yeah, you're right. that was the issue that i was having in one of the tests passing. i was forgetting to capture the output of jq. (resolved in latest commit, will re-request review once all tests pass).

i totally agree. especially when the tests are essentially more work than the original contribution...but i'd rather have too many tests and get a review suggesting to remove them, rather than too little and get the contribution blocked as has happened to me before in other opensource projects.

to be honest, i simply grepped for every instance of "--noout" and just added a test in those files. the right thing to do would be to update those tests, but i didn't want to collide with that logic since it might actually be in use somewhere, and there's a PR (#17283) that looks like it's also messing around with silencing command output.

Reply:
fixed.